homeassistant/components/sensor/homematicip_cloud.py

```diff
@@ -2,14 +2,14 @@
 Support for HomematicIP sensors.
 
 For more details about this component, please refer to the documentation at
-https://home-assistant.io/components/homematicip/
+https://home-assistant.io/components/homematicip_cloud/
```

Comment:
This url is still not correct. Look at other sensor platforms for the correct format.

Reply:
Hmm, it looks like im still not 100% used to the git branch flow, as I already started to implement new devices as well... sorry.

Reply:
Got it.